templates/3c_generate_qc_metrics.ipynb

```diff
@@ -164,96 +164,34 @@
     "\n",
```

Comment:
<p>We should update this to include a run that has more of a decrease over time, to show what level of decrease is expected. That can be a separate PR to just update the image. Will you make an issue? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>